docs/build_and_load.md

```diff
@@ -823,14 +822,22 @@ phg create-maf-vcf \
     --maf-dir output/alignment_files \
     -o output/vcf_files
 ```
-
+3, Create hVCF from existing PHG created gVCF files
+```shell
+phg gvcf2hvcf \
```

Comment:
Should this have the same parameter values as above for consistency?

Reply:
@tcasstevens - which above parameters?  Do you mean the general example on line 73, or the more detailed example for create-maf-vcf directly above?  I basically followed what was shown for create-maf-vcf above, but the gvcf folder is different as it is on the user had previously.

I'm happy to change this however you want, I'm just not clear on what you are recommending.